src/site/content/en/blog/handwriting-recognition/index.md

```diff
@@ -0,0 +1,426 @@
+---
+title: Recognize your users' handwriting
+subhead:
+  The Handwriting Recognition API allows you to recognize text from handwritten input in real time.
+description: |
+  The Handwriting Recognition API allows web applications to make use of advanced handwriting recognition services to recognize text from hand drawings in real time.
+authors:
+  - christianliebel
+  - thomassteiner
+
+date: 2021-05-14
+# updated: 2021-05-14
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/k1gdvpBMneFVrOC5h4yQ.jpg
+alt: A hand draws letters in calligraphic script on a paper.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+
+origin_trial:
+  url: https://developer.chrome.com/origintrials/#/view_trial/🎉TODO_UPDATE🎉
+---
+
+{% Aside %} The Handwriting Recognition API is part of the [capabilities project](/fugu-status/) and
+is currently in development. This post will be updated as the implementation progresses.
+{% endAside %}
+
+## What is the Handwriting Recognition API? {: #what }
+
+> TODO: GIF of demo
```

Comment:
Note that we have a `{% Video %}`. Just pass me the `.mp4`, I'll take care of uploading.